projects/gis/src/app/login/login.component.scss

```diff
@@ -8,8 +8,12 @@
 }
 
 .login {
-    margin: 0 auto;
     width: 24rem;
+
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
it's better because it works. previously the login was centered horizontally but not vertically. I am sure you can do it in a different way, but this works.